services/static-webserver/client/source/class/osparc/dashboard/ListButtonPlaceholder.js

```diff
@@ -26,31 +26,60 @@ qx.Class.define("osparc.dashboard.ListButtonPlaceholder", {
     // make unselectable
     this.addListener("changeValue", () => this.setValue(false), this);
 
+    this.__layout = this.getChildControl("progress-layout")
```

Comment:
I don't think you need to create the private member here. Just ``this.getChildControl("progress-layout")`` when you want to add the subitems to the layout, same as in the ``GridButtonBase``